### DISCUSSION TITLE
Upgrading Docker on Synology NAS

### DISCUSSION THREAD

**Discussion by optimisticzebra** [stashapp/stash#2776] [Q&A] [answered]
Hello,
I tried searching around but wasn't able to find a guide on how to update to the newest version of stash in a docker environment on a Synology NAS. Does anyone have any advise? Thanks.

**Comment by bnkai** [comment, marked as answer]
This is more of a synology issue than stash's.
Just stop the stash docker container and follow a relevant guide ( no synology available so can't verify) eg <https://www.wundertech.net/how-to-update-docker-containers-on-a-synology-nas/> 
One thing you may need to check is that you used one of the `latest` or `development` tags for stash from here <https://hub.docker.com/r/stashapp/stash/tags> and not the numbered (fixed) versions. ( fixed versions won't have an update available)

**Comment by baloodusudouest** [comment]
Hi,

I just you to use this tuto for install watchtower which update your countainer every X min or hour

https://mariushosting.com/synology-30-second-watchtower-install-using-task-scheduler-docker/
